oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/http/converter/OAuth2AccessTokenResponseHttpMessageConverter.java

```diff
@@ -55,9 +56,9 @@
 
 	private GenericHttpMessageConverter<Object> jsonMessageConverter = HttpMessageConverters.getJsonMessageConverter();
 
-	protected Converter<Map<String, String>, OAuth2AccessTokenResponse> tokenResponseConverter = new MapOAuth2AccessTokenResponseConverter();
```

Comment:
I have omitted the accessors for now, given that the `HttpMessageConverter` doesn't have one and the only use would be to override `readInternal`/`writeInternal` entirely. Since the converters are just classes, creating your own at need seems trivial enough. Let me know if you have additional thoughts on this.